addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiClient.java

```diff
@@ -50,6 +50,16 @@
     @JsonAdapter(UniFiTimestampDeserializer.class)
     protected Calendar lastSeen;
 
+    protected boolean blocked;
+
+    public boolean getBlocked() {
```

Comment:
These methods should be put in the right order not just in between fields.

Reply:
Also rename:
```suggestion
    public boolean isBlocked() {
```

Reply:
Renamed.